src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,38 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private final K[] keys = (K[])new Object[MAX_LENGTH];
+    private final V[] values = (V[])new Object[MAX_LENGTH];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
0 is default value of int